docs/_templates/include/schedule2017.rst

```diff
@@ -32,20 +32,10 @@
         <div class="col-xs-2">
           <p>{{ talk.Time }}</p>
         </div>
-        <div class="col-xs-10>
-          <p class=">
+        <div style="color: black;" class="col-xs-10">
+          <p>
 
-          {% if talk.Session in dont_link %}
```

Comment:
Because we don't have the data in the JSON using this method, so they don't work anyway. We were preprocessing the JSON in the conf.py, but that isn't happening in this method.